src/app/zap-templates/templates/chip/python-chip-ChipCluster.zapt

```diff
@@ -42,7 +42,6 @@ class ChipCluster:
     def Cluster{{asCamelCased clusterName false}}_Command{{asCamelCased name false}}(self, device: ctypes.c_void_p, ZCLendpoint: int, ZCLgroupid: int{{#chip_server_cluster_command_arguments}}, {{asCamelCased label}}: {{asPythonType chipType}}{{/chip_server_cluster_command_arguments}}):
 {{#chip_server_cluster_command_arguments}}
 {{#if (isByteString type)}}
-        {{asCamelCased label}} = {{asCamelCased label}}.encode("utf-8") + b'\x00'
```

Comment:
If this need to be an empty `{{#if}}` block, maybe we you can remove it completely ?

Reply:
Added a new helper `isCharString` for this.